modules/utxo-lib/src/bitgo/wallet/Unspent.ts

```diff
@@ -0,0 +1,70 @@
+import { Network } from '../../networkTypes';
+import { UtxoTransactionBuilder } from '../UtxoTransactionBuilder';
+import { createOutputScript2of3, scriptTypeForChain } from '../outputScripts';
+import { toOutputScript } from '../../address';
+import { signInput2Of3, verifySignatureWithPublicKeys } from '../signature';
+import { WalletUnspentSigner } from './WalletUnspentSigner';
+import { RootWalletKeys } from './WalletKeys';
+import { UtxoTransaction } from '../UtxoTransaction';
+import { Triple } from '../types';
+import { toOutput, Unspent } from '../Unspent';
+
+export interface WalletUnspent {
+  id: string;
+  address: string;
+  value: number;
+  index: number;
+  chain: number;
```

Comment:
Would it make to sense to use a type from `Codes` in `unspents` so we're restricting this to one of the known/valid chains?

Reply:
Sadly we are not really restricting it anyway, since we are not parsing the WP result (yet) (see https://github.com/gcanti/io-ts/blob/master/index.md). We basically just cast the WP response without actually validating it. 

Let's do it and see how it goes.

Reply:
done